carto/exceptions.py

```diff
@@ -10,10 +10,27 @@
 
 
 """
-
+from requests import codes
 
 class CartoException(Exception):
     """
     Any Exception produced by carto-python should be wrapped around this class
     """
     pass
+
+
+class CartoRateLimitException(CartoException):
+    def __init__(self, response):
```

Comment:
docs please:

- what's this class for
- what attributes it includes
- maybe a link to the help center with more info could be handy
- etc.

Also let's take a look at the doc folder (and generated docs). There's a section about [Exceptions](https://carto-python.readthedocs.io/en/latest/general_concepts.html#exceptions) we may want to update.